pkg/manager/member/tidbcluster_status_manager.go

```diff
@@ -120,6 +124,23 @@ func (tcsm *TidbClusterStatusManager) syncDashboardMetricStorage(tc *v1alpha1.Ti
 	return nil
 }
 
+func (tcsm *TidbClusterStatusManager) syncAutoScalerRef(tc *v1alpha1.TidbCluster) error {
+	if tc.Status.AutoScaler == nil {
+		return nil
+	}
+	tacNamespace := tc.Status.AutoScaler.Namespace
+	tacName := tc.Status.AutoScaler.Name
+	_, err := tcsm.cli.PingcapV1alpha1().TidbClusterAutoScalers(tacNamespace).Get(tacName, metav1.GetOptions{})
```

Comment:
why not use lister?

Reply:
+1, ever raised the same comment for TidbMonitor [here](https://github.com/pingcap/tidb-operator/pull/2424#discussion_r422982794)

Reply:
good catch.

Reply:
updated, and also add the unit test